Client/Frontend/Browser/TabManager.swift

```diff
@@ -194,7 +194,7 @@ class TabManager: NSObject {
         } else {
             _selectedIndex = -1
         }
-        assert(_selectedIndex > -1, "Tab expected to be in `tabs`")
+//        assert(_selectedIndex > -1, "Tab expected to be in `tabs`")
```

Comment:
For some reason, this line didn't make the feature work as expected (to be researched)

Reply:
If not needed then we could just get rid of it @noorhashem